cpp/src/arrow/compute/kernels/codegen_internal.h

```diff
@@ -181,8 +180,7 @@ struct GetOutputType<Type, enable_if_has_c_type<Type>> {
 };
 
 template <typename Type>
-struct GetOutputType<
-    Type, enable_if_t<is_string_like_type<Type>::value>> {
+struct GetOutputType<Type, enable_if_t<is_string_like_type<Type>::value>> {
   using T = std::string;
 };
```

Comment:
What is going on with all the code reformatting in this patch?

Reply:
I don't know; very odd. It appears to have only affected this file. `clang-format-8` is being used both locally and in CI

Reply:
@wesm found it: `codegen**_internal**.h` is excluded from the `format` target, so it hasn't been touched by clang-format until now. We need a JIRA for this

Reply:
https://issues.apache.org/jira/browse/ARROW-9120

Reply:
Fixed.